scheduler/src/cook/mesos/api.clj

```diff
@@ -2205,7 +2205,7 @@
     (if (nil? pool)
       (let [pool-limits (get-pool-limits get-limit-fn db user)]
         (cond-> response
-          (not (empty? pool-limits)) (assoc :pools pool-limits)))
+                (not (empty? pool-limits)) (assoc :pools pool-limits)))
```

Comment:
`cond->` conditions and clauses should be indented one level. You can change Cursive settings to fix this.

Reply:
Change reverted.